website/siteConfig.js

```diff
@@ -39,7 +39,7 @@ const siteConfig = {
     secondaryColor: "#664100"
   },
 
-  customDocsPath: "website/target/docs",
+  customDocsPath: "./docs",
```

Comment:
This diff should be reverted, the `website/target/docs` directory is generated by the sbt build.

```suggestion
  customDocsPath: "website/target/docs",
```